### DISCUSSION TITLE
Is there a way to pull in Todoist tasks to a board?

### DISCUSSION THREAD

**Discussion by thesamim** [roovo/obsidian-card-board#178] [Q&A]
Is there a way to pull Todoist tasks (from the [Todoist Plugin](https://github.com/jamiebrynes7/obsidian-todoist-plugin)) into a date or tag board?

**Comment by roovo** [comment]
Hi - I don't think that there is (using the Todoist Plugin anyway) as it looks like it gives you a view of your Todoist tasks inside Obsidian rather than importing them into the markdown documents in your vault, and this is where CardBoard reads tasks from.

Looking at the Todoist plugins, the only one that looks like it actually imports tasks and saves them in markdown in the vault is [Todoist Text](https://github.com/wesmoncrief/obsidian-todoist-text), but obviously this works differently to the Todoist Plugin so may well not be what you need.

**Comment by thesamim** [comment]
Thanks @roovo .
